### PR TITLE
Removed yaml parsing of config to initialize tracer, along with json

### PR DESCRIPTION
**Pull request by oibe**
schema validation.

### CODE REVIEW

**Comment by oibe** [comment]
fixes https://github.com/uber/jaeger-client-node/issues/40

File: src/configuration.js [hunk 1]
```diff
@@ -31,47 +31,8 @@ import LoggingReporter from './reporters/logging_reporter';
...
-    'required': ['jaeger'],
-    'properties': {
-        'jaeger': {
-            'type': 'object',
-            '$ref': '/jaeger'
-        }
-    },
-};
-
-let jaegerSchema = {
```
Comment (yurishkuro) [inline src/configuration.js]: why remove this?  My proposal in another review was to keep the schema definition, just not do the validation (only do it in tests).

Comment (oibe) [inline src/configuration.js]: what's the point of doing in tests, but not in prod?

Comment (yurishkuro) [inline src/configuration.js]: (a) it provides an authoritative schema for the config (not just a file, but the actual data structure used in the code)
(b) it lets users do validation if they choose to

**Pull request closed without merging**